docs/contracts/v4/reference/core/libraries/TickMath.md

```diff
@@ -118,7 +118,7 @@ function getSqrtPriceAtTick(int24 tick) internal pure returns (uint160 sqrtPrice
 
 ### getTickAtSqrtPrice
 
-Calculates the greatest tick value such that getSqrtPriceAtTick(tick) <= sqrtPriceX96
+Calculates the greatest tick value such that getSqrtPriceAtTick(tick) \<= sqrtPriceX96
```

Comment:
flagging for myself -- the markdown files in `v4/reference/*` are generated with `forge doc` and copied to this repo's directory structure in `uniswap/docs/scripts/v4-forge-doc.sh`

I'll update the script and use sed to make sure we're escaping docusaurus v3 special characters

---

this shouldnt block our upgrade to docusaurus v3 tho

Reply:
Could i get a ✅